### PR TITLE
fix: various UX tweaks for 07 demo

### PR DESCRIPTION
**Pull request by anandaroop**
The type of this PR is: **Fix**

Grab-bag of UX fixes for the 07 demo, including:

- un-blah-ing the page title
- following the results mockup more closely
- improving the budget field behavior, and make it optional
- preventing incomplete form submissions which lead to errors or empty rails

Screenshot 2024-06-28 at 9 01 58 AM

### CODE REVIEW

File: src/Apps/ArtAdvisor/07-Curated-Discovery/Components/Result/ArtworksRail.tsx [hunk 1]
```diff
@@ -71,7 +71,7 @@ export const ArtworksRail: FC<ArtworksRailProps> = props => {
       {artworks.length ? (
         <Box opacity={isLoading ? 0.2 : 1}>
           <Rail
-            title="Artworks"
+            title="We think you’ll be interested in these works"
```
Comment (anandaroop) [inline src/Apps/ArtAdvisor/07-Curated-Discovery/Components/Result/ArtworksRail.tsx:74]: May conflict with the WIP on the Artwork rail, but if rebasing this title was the only change.

Comment (anandaroop) [inline src/Apps/ArtAdvisor/07-Curated-Discovery/Components/Result/ArtworksRail.tsx:74]: May conflict with the WIP on the Artwork rail, but if rebasing this title was the only change.

File: src/Apps/ArtAdvisor/07-Curated-Discovery/Components/Form/Form.tsx [hunk 2]
```diff
@@ -65,6 +67,23 @@ export const Form: React.FC<FormProps> = props => {
 
   const handleSubmit = async () => {
     try {
+      if (_.isEmpty(state.goal)) {
+        sendToast({
+          variant: "error",
+          message: "Please select a goal",
```
Comment (anandaroop) [inline src/Apps/ArtAdvisor/07-Curated-Discovery/Components/Form/Form.tsx:73]: Maybe debatable, but for demo purposes: goal and interest are required; budget is optional.

Toasts as follows:

Screenshot 2024-06-28 at 9 05 10 AM

**Review by mc-jones — APPROVED** [review]
👍

**Pull request merged**